### PR TITLE
PAYARA-1845 move configuration of asadmin recorder to domain from server

### PR DESCRIPTION
**Pull request by michaelranaldo**

### CODE REVIEW

**Comment by michaelranaldo** [comment]
Jenkins test please

File: appserver/admingui/payara-console-extras/src/main/resources/asadminRecorder/asadminRecorderTabs.jsf [hunk 1]
```diff
@@ -44,16 +44,9 @@
     />
     
     <!command
-        setSessionAttribute(key="serverInstTabs" value="asadminRecorderConfig");
-        gf.redirect(page="#{request.contextPath}/payaraExtras/asadminRecorder/asadminRecorderServer.jsf?target=server&tabSetName=serverInstTabs");
+        setSessionAttribute(key="domainTabs" value="asadminRecorderConfig");
+        gf.redirect(page="#{request.contextPath}/payaraExtras/asadminRecorder/asadminRecorderServer.jsf?target=server&tabSetName=domainTabs");
     />
-
-    <sun:tab id="asadminRecorderConfig" immediate="$boolean{true}" text="$resource{i18nar.asadminRecorder.configuration}" toolTip="$resource{i18nar.asadminRecorder.configurationTip}" >
```
Comment (Pandrex247) [inline appserver/admingui/payara-console-extras/src/main/resources/asadminRecorder/asadminRecorderTabs.jsf]: Can you remove the _asadminRecorder.configuration_ and _asadminRecorder.configurationTip_ Strings from the _Strings.properties_ file please? They're no longer needed.

**Comment by Pandrex247** [comment]
jenkins test please

**Comment by michaelranaldo** [comment]
Jenkins test please

**Comment by michaelranaldo** [comment]
Jenkins test please

**Pull request merged**